configure.ac

```diff
@@ -289,7 +289,7 @@ AS_IF([test x"$with_tests_bundle_path" != x""],
       [])
 
 #Kernel path for Clear Containers
-DEFAULT_KERNEL_PATH=$localstatedir/lib/$PACKAGE_NAME/data/kernel/vmlinux.container
+DEFAULT_KERNEL_PATH=$localstatedir/lib/$PACKAGE_NAME/data/kernel/vmlinuz.container
```

Comment:
I appreciate this isn't being merged into `master`, but this change would break `master`.